crates/net/discv5/src/config.rs

```diff
@@ -35,29 +38,34 @@ pub struct DiscV5ConfigBuilder {
     /// Interval in seconds at which to run a lookup up query with local node ID as target, to
     /// populate kbuckets.
     self_lookup_interval: Option<u64>,
+    /// Optional filter rules to apply to a discovered peer in order to determine if it should be
+    /// passed up to mempool or dropped.
```

Comment:
mempool?

Reply:
yeah, peers are discovered by discovery protocol, in order to connect to them over a mempool session.